src/EventManager.ts

```diff
@@ -27,13 +28,15 @@ export class EventManager {
 	 *   // event.pos
 	 *   // event.input
 	 *   // event.inputEvent
+	 *   // isTrusted
 	 * });
 	 */
 	triggerHold(pos: Axis, input: IInputType, event) {
 		this.axes.trigger("hold", {
 			pos,
 			input,
 			inputEvent: event,
+			isTrusted: !!event,
```

Comment:
Is it possible that `isTrusted` is `false`? If not, If not, is it better to use constant(`true`) to reduce considering means.